swagger/public_v1/component_schemas/ProviderFilter.yml

```diff
@@ -2,6 +2,39 @@ type: object
 example: ""
 description: "This schema is used to search within collections to return more specific results."
 properties:
+  can_sponsor_skilled_worker_visa:
+    description: "Only return providers that can sponsor skilled worker visa."
+    type: boolean
+    example: true
+  can_sponsor_student_visa:
+    description: "Only return providers that can sponsor student visa."
+    type: boolean
+    example: true
+  provider_type:
+    description: "Return providers based on their provider type. This is a comma delimited string. If multiple provider types are provided then any provider matching any one of the options provider will be returned, i.e. the OR operator is used."
+    type: string
+    example: "scitt,lead_school"
+    enum:
+      - scitt
+      - lead_school
+      - university
```

Comment:
```suggestion
      - lead_school
      - scitt
      - university
```

Reply:
Done in https://github.com/DFE-Digital/publish-teacher-training/pull/3150/commits/ff2419087f65a61e6095a235b6f3ff4486042bee